scripts/release_scripts/update_changelog_and_credits_test.py

```diff
@@ -503,15 +503,17 @@ def test_create_branch(self):
             'run_cmd_gets_called': True,
             'open_new_tab_in_browser_if_possible_gets_called': True
         }
+        filepaths_get_contents_is_called_with = []
         def mock_get_branch(unused_self, unused_branch_name):
             check_function_calls['get_branch_gets_called'] = True
             return github.Branch.Branch(
                 requester='', headers='',
                 attributes={'commit': {'sha': 'test'}}, completed='')
         def mock_create_git_ref(unused_self, ref, sha):  # pylint: disable=unused-argument
             check_function_calls['create_git_ref_gets_called'] = True
-        def mock_get_contents(unused_self, unused_filepath, ref):  # pylint: disable=unused-argument
+        def mock_get_contents(unused_self, filepath, ref):  # pylint: disable=unused-argument
```

Comment:
Do we still need param?

Reply:
What do you mean by this?

Reply:
Sorry for the confusion I've updated the comment PTAL!

Reply:
It is still needed for the `ref` arg.